packages/parse-markdown-table/lib.ts

```diff
@@ -10,7 +10,7 @@ export interface TableIterationOptions {
   readonly rightBound: boolean
 }
 
-const horizontalLineRegex = /^[|-]+$/
+const horizontalLineRegex = /^[|\- ]+$/
```

Comment:
Does this mean that it would match a line containing only space characters?

Reply:
I think you need to fix this regex.

Reply:
Oh yeah that could be a problem. I will make the regex better and add a test to make sure a "line containing only space characters" does not count as a horizontal line.

Reply:
@KSXGitHub I don't think matching an empty character will change any current behavior.

Example:
```md
| id | name        | email                    |
    
|  1 | John Doe    | john-doe@gmail.com       |
|  2 | Peter Smith | petersmith22@outlook.com |
|  3 | Julia Jones | jjones778@gmail.com      |
```
This table is currently parsed as a valid table (on the `master` branch, and on this PR).

So changing the `horizontalLineRegex` to not work with whitespace lines won't change any behavior. The only line it effects is [`    if (horizontalLineRegex.test(trimmedLine)) continue`](https://github.com/ksxnodeapps/fun-little-utilities/blob/9f96674f517db5d1ca5efe00913a13c8524147a5/packages/parse-markdown-table/lib.ts#L20), which doesn't affect the output.